src/test/resources/testing-editor.md

```diff
@@ -218,12 +231,16 @@ metadata:
 
 -> Error notification appears
 
+COMMENT: is a part of previous, don't need to test same thing twice
+
 **Error notification appears on startup**
 1. have editor with invalid content
 2. restart IJ
 
 -> Error notification appears
 
+NOTE: "Edit..." some resource -> paste "   " before "---" -> see error notification -> restart IDE -> error notification disappears. Works well with "aaa" after "metadata:".
```

Comment:
The problem here is that after the restart, the editor content is NOT detected as containing a kubernetes resource. The content is NOT valid kubernetes yaml. The editor therefore has NO toolbar and therefore there's no error reported. This only becomes a kubernetes editor as soon as the content gets valid. From then on you have a toolbar and any change gets notified prompting you to push, pull or by erroring.
﻿
<img width="536" alt="grafik" src="https://user-images.githubusercontent.com/25126/175322240-37cbd817-522f-4fae-9c8b-ba24942c0f67.png">